### PR TITLE
footer updates, team members & photos, logo update, and i probably bu…

### PR DESCRIPTION
**Pull request by kenyatta**
…rned something to the ground in the process #srrynotsrry.

### CODE REVIEW

**Comment by sbLL** [comment]
nope

File: _data/projects/lovely-direct.yml [hunk 1]
```diff
@@ -5,11 +5,13 @@ github:
 - TeamLovely/leasing-public
 - TeamLovely/leasing-private
 - TeamLovely/leasing-common
-description: With Lovely Direct, we do all the legwork and you just sit back and collect the rent checks. It's that easy. From marketing and pricing to open houses and applicant screening, we're a landlord's best friend.
-milestones: 
+description: At the heart of Lovely Direct is the desire to simplify renting in a new way for both sides of the marketplace. Landlord want to lease the properties quickly and with minimal effort, while leasing their units for a fair price. Renters want to have a smooth and transparent experience free of scams, unresponsive communication, and hidden fees. To accomplish this, Lovely Direct is offering a full service solution to give us control of  inventory so that we will be able to make promises to both landlords and renters. <br /><br />For landlord we can ensure proposing qualified renters within a given time window, through our individualized listing creations, data driven pricing, and thorough screenings. For renters, we can ensure efficient response rates, clearly communicate all the requirements, and secure and simplify the application process.
```
Comment (coderkat) [inline _data/projects/lovely-direct.yml]: typo: 'Landlords' or 'a landlord', right?

**Comment by coderkat** [comment]
thanks for that photo, @kenyatta

**Comment by coderkat** [comment]
:+1:

**Pull request merged**